### PR TITLE
Update mqtt_room markdown

### PR DESCRIPTION
**Pull request by jptrsn**
**Description:**
Updated list of applications that work with mqtt room component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/mqtt_room.markdown [hunk 1]
```diff
@@ -72,5 +72,6 @@ Example JSON that should be published to the room topics:
 This component works with any software that is sending data in the given format. Each client should post the discovered devices in its own subtopic of the configured topic.
 Instead of developing your own application, you can also use any of these already existing clients:
 
-- [**room-assistant**](https://github.com/mKeRix/room-assistant): looks for Bluetooth LE beacons, based on Node.js
+- [**room-assistant**](https://github.com/mKeRix/room-assistant): runs on a raspberry pi, and looks for Bluetooth LE beacons, based on Node.js
```
Comment (frenck) [inline source/_components/mqtt_room.markdown]: This is a bit weird since it might suggest it only runs on a Raspberry Pi?

**Review by frenck — CHANGES_REQUESTED** [review]
One small question. Could you please take a look @jptrsn? 👍

File: source/_components/mqtt_room.markdown (hunk 1 above)
Comment (jptrsn) [reply source/_components/mqtt_room.markdown]: Good point. Removed change in subsequent commit.

**Review by klaasnicolaas — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for the PR and the quick update @jptrsn!

**Pull request merged**